docs/kratos/social-signin/09_ory.mdx

```diff
@@ -0,0 +1,182 @@
+---
+id: ory
+title: Add Ory OAuth2 or Ory Hydra as a social sign-in provider
+sidebar_label: Ory
+---
+
+# Ory
+
+This **Ory** option allows you to add Ory OAuth2 as OIDC provider. In Ory Network, you can create this provider using the Ory
+Console or the Ory CLI. When self-hosting Ory, deploy the Ory Hydra Federation server and use the `hydra` CLI to create the
+client.
```

Comment:
Let's explain the setup upfront:
- We create one Ory project that serves as the SSO provider
- We show how to wire _another_ Ory project - e.g. in a 3rd party app - so that it uses this SSO provider.